Packs/FeedZeroDayLiveTIFusion/pack_metadata.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "Zero Day Live TI FUSION Feed",
+    "description": "'Zero Day Live is Blackwired’s flagship product that delivers proprietary",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Blackwired Ltd.",
```

author	-	The name of the organization (for partners) or developer (for individual contributions) which developed the integration.

```suggestion
    "author": "Grzegorz Papkala",
```